skits/pipeline.py

```diff
@@ -228,11 +228,11 @@ def predict(self, X, to_scale=False, refit=True, start_idx=0):
                 prediction[:, [idx]] = self.inverse_transform(prediction[:, [idx]])
         return prediction[start_idx:].squeeze()
 
-    def forecast(self, X, start_idx, trans_window=None):
+    def forecast(self, X, start_idx, forecast_window=None, trans_window=None):
```

Comment:
Just realized, this is one of the few functions where I actually have docstrings... Can you add an explanation of `forecast_window`?

Reply:
Makes perfect sense, I will go ahead and implement these changes